toonz/sources/stdfx/igs_line_blur.cpp

```diff
@@ -3026,7 +3026,7 @@ void pixel_line_root::exec09_same_way_expand(
                        clp_select->get_i32_count_max());
   }
 
-  /* 各端点(one,anohter)ごと
+  /* 各端点(one,another)ごと
```

Comment:
Another automated typo fix that leaves the word typo'd in yet another form.
See also line 3052.